### DISCUSSION TITLE
ReEDS is now open-source!

### DISCUSSION THREAD

**Discussion by wesleyjcole** [NREL/ReEDS-2.0#156] [Announcements]
The ReEDS model was previously an open-access model in a private repository.  The model has now been made open-source in a public repository under the GNU GPL v3.0 license.  We hope you enjoy the expanded opporutnity to use the model!

**Comment by gschivley** [comment]
Thanks Wes, it's great to see ReEDS move to a public repo with a standard open-source license! Do you have any sense how this is supposed to affect the underlying data? The data is packaged with the repo and not available elsewhere but I don't know if it is supposed to count as part of the "covered work". Based on the federal government [open data policy](https://resources.data.gov/open-licenses/) I would assume that reproduction and redistribution of the data should be allowed without the restrictions that GPL v3 would require.

**Reply by wesleyjcole** [reply]
This question is definitely outside of my ability to answer.  I'll ask around and see what I can learn.
